vbox: TestAddons/parallel/Ingress: pods "app.kubernetes.io/name=nginx-ingress-controller": timed out waiting for the condition

```
        --- FAIL: TestAddons/parallel/Ingress (189.30s)
            addons_test.go:75: (dbg) Run:  out/minikube-linux-amd64 -p addons-20190913T004504.49137044-19644 addons enable ingress
            addons_test.go:88: Waiting for pods with labels "app.kubernetes.io/name=nginx-ingress-controller" in namespace "kube-system" ...
            helpers.go:239: "nginx-ingress-controller-784df4b8bb-4zlzw" [21e53a50-cc67-4dd8-9ad8-cacf16d81a8b] Pending
            helpers.go:239: "nginx-ingress-controller-784df4b8bb-4zlzw" [21e53a50-cc67-4dd8-9ad8-cacf16d81a8b] Pending / Ready:ContainersNotReady (containers with unready status: [nginx-ingress-controller]) / ContainersReady:ContainersNotReady (containers with unready status: [nginx-ingress-controller])
            addons_test.go:88: pods "app.kubernetes.io/name=nginx-ingress-controller": timed out waiting for the condition
            helpers.go:284: (dbg) Run:  kubectl --context addons-20190913T004504.49137044-19644 get po -A --show-labels
            helpers.go:290: (dbg) kubectl --context addons-20190913T004504.49137044-19644 get po -A --show-labels:
                NAMESPACE     NAME                                        READY   STATUS              RESTARTS   AGE    LABELS
                default       registry-test                               0/1     ContainerCreating   0          49s    run=registry-test
                kube-system   coredns-6fb7db48b4-mf2vf                    1/1     Running             1          21m    k8s-app=kube-dns,pod-template-hash=6fb7db48b4
                kube-system   coredns-6fb7db48b4-mrmxk                    1/1     Running             1          21m    k8s-app=kube-dns,pod-template-hash=6fb7db48b4
                kube-system   etcd-minikube                               1/1     Running             0          21m    component=etcd,tier=control-plane
                kube-system   kube-addon-manager-minikube                 1/1     Running             0          20m    component=kube-addon-manager,kubernetes.io/minikube-addons=addon-manager,version=v9.0
                kube-system   kube-apiserver-minikube                     1/1     Running             0          21m    component=kube-apiserver,tier=control-plane
                kube-system   kube-controller-manager-minikube            1/1     Running             0          21m    component=kube-controller-manager,tier=control-plane
                kube-system   kube-proxy-xbdq8                            1/1     Running             0          21m    controller-revision-hash=7c7847858f,k8s-app=kube-proxy,pod-template-generation=1
                kube-system   kube-scheduler-minikube                     1/1     Running             0          21m    component=kube-scheduler,tier=control-plane
                kube-system   nginx-ingress-controller-784df4b8bb-4zlzw   0/1     ContainerCreating   0          3m1s   addonmanager.kubernetes.io/mode=Reconcile,app.kubernetes.io/name=nginx-ingress-controller,app.kubernetes.io/part-of=kube-system,pod-template-hash=784df4b8bb
                kube-system   registry-f5wpr                              1/1     Running             0          3m1s   actual-registry=true,addonmanager.kubernetes.io/mode=Reconcile,kubernetes.io/minikube-addons=registry
                kube-system   registry-proxy-mjlll                        1/1     Running             0          3m1s   addonmanager.kubernetes.io/mode=Reconcile,controller-revision-hash=5457c46479,kubernetes.io/minikube-addons=registry,pod-template-generation=1,registry-proxy=true
                kube-system   storage-provisioner                         1/1     Running             0          21m    addonmanager.kubernetes.io/mode=Reconcile,integration-test=storage-provisioner
            helpers.go:293: (dbg) Run:  kubectl --context addons-20190913T004504.49137044-19644 describe po nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system
            helpers.go:297: (dbg) kubectl --context addons-20190913T004504.49137044-19644 describe po nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system:
                Name:           nginx-ingress-controller-784df4b8bb-4zlzw
                Namespace:      kube-system
                Priority:       0
                Node:           minikube/10.0.2.15
                Start Time:     Fri, 13 Sep 2019 01:12:37 -0700
                Labels:         addonmanager.kubernetes.io/mode=Reconcile
                                app.kubernetes.io/name=nginx-ingress-controller
                                app.kubernetes.io/part-of=kube-system
                                pod-template-hash=784df4b8bb
                Annotations:    prometheus.io/port: 10254
                                prometheus.io/scrape: true
                Status:         Pending
                IP:             
                Controlled By:  ReplicaSet/nginx-ingress-controller-784df4b8bb
                Containers:
                  nginx-ingress-controller:
                    Container ID:  
                    Image:         quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.25.1
                    Image ID:      
                    Ports:         80/TCP, 443/TCP, 18080/TCP
                    Host Ports:    80/TCP, 443/TCP, 18080/TCP
                    Args:
                      /nginx-ingress-controller
                      --configmap=$(POD_NAMESPACE)/nginx-load-balancer-conf
                      --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
                      --udp-services-configmap=$(POD_NAMESPACE)/udp-services
                      --annotations-prefix=nginx.ingress.kubernetes.io
                      --report-node-internal-ip-address
                    State:          Waiting
                      Reason:       ContainerCreating
                    Ready:          False
                    Restart Count:  0
                    Liveness:       http-get http://:10254/healthz delay=10s timeout=1s period=10s #success=1 #failure=3
                    Readiness:      http-get http://:10254/healthz delay=0s timeout=1s period=10s #success=1 #failure=3
                    Environment:
                      POD_NAME:       nginx-ingress-controller-784df4b8bb-4zlzw (v1:metadata.name)
                      POD_NAMESPACE:  kube-system (v1:metadata.namespace)
                    Mounts:
                      /var/run/secrets/kubernetes.io/serviceaccount from nginx-ingress-token-c8vpl (ro)
                Conditions:
                  Type              Status
                  Initialized       True 
                  Ready             False 
                  ContainersReady   False 
                  PodScheduled      True 
                Volumes:
                  nginx-ingress-token-c8vpl:
                    Type:        Secret (a volume populated by a Secret)
                    SecretName:  nginx-ingress-token-c8vpl
                    Optional:    false
                QoS Class:       BestEffort
                Node-Selectors:  <none>
                Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                                 node.kubernetes.io/unreachable:NoExecute for 300s
                Events:
                  Type    Reason     Age        From               Message
                  ----    ------     ----       ----               -------
                  Normal  Scheduled  <unknown>  default-scheduler  Successfully assigned kube-system/nginx-ingress-controller-784df4b8bb-4zlzw to minikube
                  Normal  Pulling    2m52s      kubelet, minikube  Pulling image "quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.25.1"
            helpers.go:300: (dbg) Run:  kubectl --context addons-20190913T004504.49137044-19644 logs nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system
            helpers.go:300: (dbg) Non-zero exit: kubectl --context addons-20190913T004504.49137044-19644 logs nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system: exit status 1 (233.413347ms)
            helpers.go:300: (dbg) Command: kubectl --context addons-20190913T004504.49137044-19644 logs nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system
                
                ** stderr ** 
                Error from server (BadRequest): container "nginx-ingress-controller" in pod "nginx-ingress-controller-784df4b8bb-4zlzw" is waiting to start: ContainerCreating
                
            helpers.go:302: kubectl --context addons-20190913T004504.49137044-19644 logs nginx-ingress-controller-784df4b8bb-4zlzw -n kube-system: exit status 1
            addons_test.go:89: wait: app.kubernetes.io/name=nginx-ingress-controller within 3m0s: timed out waiting for the condition
```

It appears to be a case of not giving a long enough time for a large image pull from quay.io.

--- a/test/integration/addons_test.go
+++ b/test/integration/addons_test.go
@@ -82,10 +82,10 @@ func validateIngressAddon(ctx context.Context, t *testing.T, profile string) {
 		t.Fatalf("kubernetes client: %v", client)
 	}
 
-	if err := kapi.WaitForDeploymentToStabilize(client, "kube-system", "nginx-ingress-controller", time.Minute*5); err != nil {
+	if err := kapi.WaitForDeploymentToStabilize(client, "kube-system", "nginx-ingress-controller", 6*time.Minute); err != nil {
 		t.Errorf("waiting for ingress-controller deployment to stabilize: %v", err)
 	}
-	if _, err := PodWait(ctx, t, profile, "kube-system", "app.kubernetes.io/name=nginx-ingress-controller", 3*time.Minute); err != nil {
+	if _, err := PodWait(ctx, t, profile, "kube-system", "app.kubernetes.io/name=nginx-ingress-controller", 8*time.Minute); err != nil {
 		t.Fatalf("wait: %v", err)
 	}
 
@@ -142,15 +142,15 @@ func validateRegistryAddon(ctx context.Context, t *testing.T, profile string) {
 	}
 
 	start := time.Now()
-	if err := kapi.WaitForRCToStabilize(client, "kube-system", "registry", 3*time.Minute); err != nil {
+	if err := kapi.WaitForRCToStabilize(client, "kube-system", "registry", 6*time.Minute); err != nil {
 		t.Errorf("waiting for registry replicacontroller to stabilize: %v", err)
 	}
 	t.Logf("registry stabilized in %s", time.Since(start))
 
-	if _, err := PodWait(ctx, t, profile, "kube-system", "actual-registry=true", 3*time.Minute); err != nil {
+	if _, err := PodWait(ctx, t, profile, "kube-system", "actual-registry=true", 6*time.Minute); err != nil {
 		t.Fatalf("wait: %v", err)
 	}
-	if _, err := PodWait(ctx, t, profile, "kube-system", "registry-proxy=true", 3*time.Minute); err != nil {
+	if _, err := PodWait(ctx, t, profile, "kube-system", "registry-proxy=true", 6*time.Minute); err != nil {
 		t.Fatalf("wait: %v", err)
 	}
 

